### PR TITLE
Renaming things for Authentication with Okta

### PR DESCRIPTION
**Pull request by coldlink**
## What does this change?
- Rename `idapiMock` to `mock` in cypress tests
  - as we'll be using the same endpoint to mock Okta requests too, the name no longer makes sense
- Rename `APIFetch` to `IDAPIFetch` as that better reflects that it only deals with identity

### CODE REVIEW

File: cypress/integration/sign_in.spec.js [hunk 1]
```diff
@@ -15,10 +15,12 @@ describe('Sign in flow', () => {
     });
 
     it('Has no detectable a11y violations on sign in page with error', () => {
-      cy.visit('/signin');
+      cy.visit(
+        '/signin?error=error&error_description=oh no somethings gone wrong',
+      );
```
Comment (oliverlloyd) [inline cypress/integration/sign_in.spec.js:20]: This related?

File: cypress/support/commands.js [hunk 2]
```diff
@@ -62,13 +62,13 @@ Cypress.Commands.add('idapiMockPattern', (status, body, pattern) => {
   cy.request(getMockOptions(status, body));
 });
 
-Cypress.Commands.add('idapiLastPayloadIs', (expected) => {
+Cypress.Commands.add('lastPayloadIs', (expected) => {
```
Comment (oliverlloyd) [inline cypress/support/commands.js:65]: Are we using this command, maybe we can remove it?

File: cypress/support/commands.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ Cypress.Commands.add('idapiMockNext', (status, body) => {
   cy.request(getMockOptions(status, body));
 });
 
-Cypress.Commands.add('idapiMockAll', (status, body, path) => {
+Cypress.Commands.add('mockAll', (status, body, path) => {
```
Comment (oliverlloyd) [inline cypress/support/commands.js:23]: Are we using this command, maybe we can remove it?

File: src/server/lib/middleware/index.ts [hunk 4]
```diff
@@ -8,7 +8,7 @@ import { getConfiguration } from '@/server/lib/getConfiguration';
 import { requestStateMiddleware } from '@/server/lib/middleware/requestState';
 import { default as routes } from '@/server/routes';
 import { routeErrorHandler } from '@/server/lib/middleware/errorHandler';
-import { fourZeroFourMiddleware } from './404';
+import { fourZeroFourMiddleware } from '@/server/lib/middleware/404';
```
Comment (oliverlloyd) [inline src/server/lib/middleware/index.ts:11]: 👍

File: src/server/routes/signIn.ts [hunk 5]
```diff
@@ -9,7 +9,7 @@ import { Metrics } from '@/server/models/Metrics';
 import { PageTitle } from '@/shared/model/PageTitle';
 import { handleAsyncErrors } from '@/server/lib/expressWrappers';
 import { setIDAPICookies } from '@/server/lib/setIDAPICookies';
-import { getConfiguration } from '../lib/getConfiguration';
+import { getConfiguration } from '@/server/lib/getConfiguration';
```
Comment (oliverlloyd) [inline src/server/routes/signIn.ts:12]: 👍

**Review by oliverlloyd — APPROVED** [review]
Just a couple of small things, otherwise 🚀

File: cypress/support/commands.js (hunk 2 above)
Comment (coldlink) [reply cypress/support/commands.js:65]: it's being used in the onboarding flow spec (may need to expand the diff in github)

File: cypress/support/commands.js (hunk 3 above)
Comment (coldlink) [reply cypress/support/commands.js:23]: also being used in the onboarding flow spec (may need to expand the diff in github)

Comment (oliverlloyd) [reply cypress/support/commands.js:23]: Ah! Gotcha

File: cypress/integration/sign_in.spec.js (hunk 1 above)
Comment (coldlink) [reply cypress/integration/sign_in.spec.js:20]: ah it's not! crept in with the cherry-pick, let me change that

```suggestion
      cy.visit(
        '/signin',
      );
```

**Pull request merged**